packages/create-llama/templates/types/simple/fastapi/app/settings.py

```diff
@@ -0,0 +1,8 @@
+import os
+from llama_index.llms.openai import OpenAI
+from llama_index.core.settings import Settings
+
+
+def init_base_settings():
```

Comment:
how can call this `init_settings` and override it if we select a vector store?

Reply:
We have another settings.py file in each vectordbs template code which calls the init_base_settings and overrides the config. Basically, i just use the Settings as ServiceContext to keep our code structure as it is.